fullcalendar-app/src/config/apiConfig.js

```diff
@@ -0,0 +1,11 @@
+// apiConfig.js
+
+const API_BASE_URL = window.location.origin;
+const SCHEDULES_ENDPOINT = '/schedules/get-event-data-date-range';
+const UPDATE_EVENT_ENDPOINT = '/admin/openy/branch-schedules/update-event';
```

Comment:
can we avoid using /admin/ paths in FE apps? There are some rules in Drupal core  causing permissions issues as well as possible security downsides

Reply:
Fixed.
The admin path is saved only for the list of branches and the calendar itself, if they also need to be changed, please let me know.